torchci/components/JobLinks.tsx

```diff
@@ -301,3 +314,22 @@ function DisableIssue({
     </a>
   );
 }
+
+function RevertInfoCopy({ job }: { job: JobData }) {
+  const info = [];
+  const testName = getTestName(
+    (job.failureLines && job.failureLines[0]) ?? "",
+    false,
+    true
+  );
+  if (testName !== null) {
+    info.push(testName);
+  }
+  info.push(job.htmlUrl);
+  info.push(`https://hud.pytorch.org/${job.repo}/commit/${job.sha}`);
+  return CopyLink({
+    textToCopy: info.join(" "),
+    copyPrompt: "Revert Info",
```

Comment:
I'm wondering what a better prompt for this could be. Perhaps replace the "link" chain icon with the [copy icon](https://thenounproject.com/icon/copy-4623897/) to emphasize that you're copying text instead of a link to something

Reply:
Done, but I also changed the other icons to be icons instead of emojis, not sure if that is desirable or now